wallet_ui/components/panels/CreateCanister.tsx

```diff
@@ -71,9 +71,8 @@ export function CreateCanisterDialog(props: {
   const [cycles, setCycles] = React.useState(0);
   const [balance, setBalance] = React.useState(0);
   const [canisterId, setCanisterId] = React.useState<Principal | undefined>();
-  const [error1, setError1] = React.useState(false);
-  const [error2, setError2] = React.useState(false);
-  const [moreControllers, setMoreControllers] = React.useState<string[] | []>([]);
+  const [error, setError] = React.useState([false]);
+  const [controllersState, setControllers] = React.useState<string[] | []>([walletPrincipal]);
```

Comment:
For conventions' sake, it's best to have the value use the same name as the setter

```suggestion
  const [controllers, setControllers] = React.useState<string[] | []>([walletPrincipal]);
```